ui.frontend/src/view/FormDatePicker.js

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright 2024 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+import FormFieldBase from "./FormFieldBase.js";
+import {parse} from "@aemforms/af-formatters";
+
+/**
+ * Class representing Date picker.
+ * @extends module:FormView~FormDatePicker
+ */
+class FormDatePicker extends FormFieldBase {
+
+    constructor(params) {
+        super(params);
+    }
+
+    updateFormattedDate(value){
+        this._model.value = this.formatDate(value);
+    }
+
+    formatDate(value) {
+        const currDate = parse(value.toString(), this._model.locale, this._model.editFormat, null, false);
```

Comment:
I am not sure what happens if the editFormat is not defined.

Reply:
the default editFormat is date|short